Tests/KarrotFlexTests/FlexCenterTests.swift

```diff
@@ -1,8 +1,17 @@
+//
+//  FlexCenterTests.swift
+//
+//
+//  Created by Kanghoon Oh on 2022/01/14.
+//
+
 import XCTest
+
+import SnapshotTesting
```

Comment:
Why did you replace to SnapshotTesting? 
https://github.com/pointfreeco/swift-snapshot-testing

Reply:
The library that supports the snapshot test is as follows.
- https://github.com/pointfreeco/swift-snapshot-testing
- https://github.com/uber/ios-snapshot-test-case

but `ios-snapshot-test-case` is difficult to use in Swift Packages (because it use Scheme options).